tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -1119,10 +1119,10 @@ public function eventCreate(array $params = []): array {
       'event_type_id' => 1,
       'is_public' => 1,
       'start_date' => 20081021,
-      'end_date' => 20081023,
+      'end_date' => '+ 1 month',
```

Comment:
so a 6 year event? :)

Reply:
@seamuslee001 yeah - key thing is the end date is in the future & start date is in the past so when you try to use it you don't get told it is invalid